src/cascade/saver/db/save_model_results_to_db.py

```diff
@@ -62,26 +110,25 @@ def _write_temp_draws_file_and_upload_model_results(draws_df, execution_context)
 
     with tempfile.TemporaryDirectory() as tmpdirname:
```

Comment:
What happens if there is an exception?  Does the temporary directory get removed?

Reply:
Yes. That is the main advantage of using these context managers. Regardless of how control exits the context, either due to successful completion or an exception, the cleanup happens as expected. Which can actually be irritating for debugging because the directory gets removed as soon as an exception happens which means it get's removed before `pdb` has a chance to stop the program and get you into the interactive debugger.

Reply:
Ok, DD mentioned that my ScratchMaker had something that this doesn't.  I was wondering if it was my clean-up.  I saw some tmp directories sticking around when I was working on this new version of the code, which was fun to see, but not sure now how I managed to do that.

Reply:
Hmm. No, short of a segfault or a problem with the file system this shouldn't leave dangling directories. But there could be something I'm not understanding.